config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  resources :restaurants, only: [:show, :new, :create]
+  resources :restaurants, only: [:show, :new, :create] do
+    resources :menus
```

Comment:
What routes are you generating here vs what routes are you using. Remember to scope down the amount of routes you generate (I've already mentioned this in the context of this project)